uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/Stats/StatsGathererTest.php

```diff
@@ -115,6 +135,10 @@ public function testEntityCountsWillMatchExpectedCount() {
     $this->assertEquals(1, $stats->getEntityCount('assignment'));
     $this->assertEquals(2, $stats->getEntityCount('document'));
     $this->assertEquals(2, $stats->getEntityCount('leaveRequest'));
+
+    $this->assertEquals(2, $stats->getEntityCount('jobRole'));
+    $this->assertEquals($costCenterCount, $stats->getEntityCount('costCenter'));
+    $this->assertEquals(2, $stats->getEntityCount('caseType'));
```

Comment:
This test keeps growing and growing with each new entity count added to the pingback payload. Would it be too difficult to have separate tests for each entity?

Reply:
Good idea, will do